docs/getting_started.md

```diff
@@ -92,6 +93,10 @@ pip install mmcv-full
 
 **d. Install [MMDetection](https://github.com/open-mmlab/mmdetection).**
 
+Note:
+
+MMDetection3D v0.13.0 is only compatiable with MMDetection version mmdet>=2.10.0, <=2.11.0.
```

Comment:
MMDetection3D v0.13.0 is only compatiable with MMDetection version mmdet>=2.10.0, <=2.11.0. The future versions will only support mmdet>=2.12.0 since the v0.14.0 (to be released in May).